src/assets/js/search.js

```diff
@@ -0,0 +1,74 @@
+(() => {
+  const getExcerpt = (content, query) => {
+    const queryRegex = new RegExp(query, "i");
+    const searchIndex = content.search(queryRegex);
+    const queryLength = query.length;
+    const excerpt = content.slice(
+      searchIndex - 100,
+      searchIndex + queryLength + 100
+    );
+
+    return excerpt.replace(
+      queryRegex,
+      "<strong class='search-keyword'>$&</strong>"
+    );
+  };
+
+  const displaySearchResults = (results, store, searchTerm) => {
+    const searchResults = document.getElementById("search-results");
```

Comment:
Having a `results` parameter and a `searchResults` constant is a little confusing - could this be renamed `searchResultsElement` or `searchResultsNode`?

Reply:
Yeah, you're right. I've named it `searchResultsElement` as that lined up more with the `getElement` function.